MAINTAINERS.md

```diff
@@ -135,3 +136,12 @@ the "General" channel of the [Urbit Community group][urbit-community].
 - [ ] `~zod` and other live galaxies (TODO which?) should all be updated to the new binary, to make sure they will be ready to install and deploy Arvo releases on the latest Kelvin.
 - [ ] `~marzod` and other live stars (TODO which?) should also be updated to the new binary.
 - [ ] Ships that host official UF groups should be updated: `~bitdeg`, `~halbex-palheb`, TODO: more.
+
+## Preparing for the Next Release
+
+This instructions are performed **on release day** to prepare the next release candidate.
+
+- [ ] Merge `master` into `develop` and `git push origin develop`
+  - `master` now has the contents of `release`, which may have deviated from `develop` were any patches or reversions made
```

Comment:
Looks like an extra space snuck in here.

Reply:
That's intentional.